test/drive_test.dart

```diff
@@ -0,0 +1,54 @@
+import 'dart:convert';
+import 'package:mockito/mockito.dart';
+import 'package:motis_mitfahr_app/drives/models/drive.dart';
+import 'package:motis_mitfahr_app/util/supabase.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'mock_server.dart';
+import 'mock_server.mocks.dart';
+
+// Die Klasse UrlProcessor muss zu Beginn jeder Testdatei implementiert werden und die Methode processUrl überschrieben werden
+// Wird die Methode ProcessUrl aufgrufen, wird für den dort definierten Fall (in dem Beispiel client.from('drives').select('driver_id,seats')) die Antwort definiert
+// Die Datenbankabfrage an den Client wird so abgefangen und das gewünschte Ergebnis zurückgegeben.
+// Um herauszufinden welche URL durch die jeweilige Datenbankabfrage generiert wird, einfach den auskommentierten Print-Aufruf in der mockServer.Dart Datei aktivieren
+
+void main() {
+  MockUrlProcessor driveProcesor = MockUrlProcessor();
+  //setup muss in jeder Testklasse einmal aufgerufen werden
+  setUp(() async {
+    await MockServer.initialize();
+    MockServer.handleRequests(driveProcesor);
+    SupabaseManager.setClient(MockServer.client);
```

Comment:
Can you put this in the MockServer.initialize method?

Reply:
good idea